### ISSUE TITLE
SENECA TERMINATED (on timeout)

### ISSUE THREAD

**Issue by balagcontus** [oddnetworks/oddworks#18]
Hi team,

When run node index.js , I am getting SENECA TERMINATED (on timeout)

Please reply.

**Comment by chiefy** [comment]
@balagcontus can you provide more information, like a stack trace or console output? Do you have Elasticsearch running and configured?

**Comment by balagcontus** [comment]
Hi,

Thanks for your answer. 
I have installed and configured elastic search successfully. 

![screenshot_2016-04-01_18-30-53](https://cloud.githubusercontent.com/assets/5197274/14207482/095f7154-f838-11e5-86a4-d7c1442f8f70.png)

**Comment by chiefy** [comment]
@balagcontus are you running mongodb? have you set your environment up proper? Are you running in development or production mode?

**Comment by balagcontus** [comment]
@chiefy I have changed to development mode  and get different error, please look screenshot
![screenshot_2016-04-01_18-35-55](https://cloud.githubusercontent.com/assets/5197274/14207595/befe098a-f838-11e5-807f-993ad354eb6b.png)
. 

Thanks for your answer.

**Comment by chiefy** [comment]
@balagcontus if you are using AWS Elasticsearch, you need to make sure you have setup proper IAM roles and then sign the requests with the proper AWS credentials, try setting 

```
AWS_ES_CREDS=username:password
AWS_DEFAULT_REGION=us-west-2
```

replace `username` with the AWS key and `password` with AWS secret key, but keep the `:` in between. Also remember to set the region.

**Comment by balagcontus** [comment]
@chiefy Thanks for your guidance.

I given open policy in AWS Elasticsearch **Allow open access to the domain** 
 
Now different error, please look attached screenshot. 

![screenshot_2016-04-01_19-34-30](https://cloud.githubusercontent.com/assets/5197274/14209271/fb12a126-f840-11e5-8233-e9ffc591d178.png)

**Comment by chiefy** [comment]
@balagcontus try deleting/unsetting the `AWS_ES_CREDS` and `AWS_DEFAULT_REGION` then otherwise the plugin will try signing all requests to AWS ES, and that may be causing the issue.

**Comment by balagcontus** [comment]
@chiefy 

Very thanks for prompt response. 

I have configured .ENV file 

ELASTICSEARCH_URI=https://search-nodejs-mwlccypaf4uj3ikjtzdz6kyhju.ap-southeast-1.es.amazonaws.com/
AWS_ES_CREDS=nodejs:nodejs123
AWS_DEFAULT_REGION=ap-southeast-1

Please give suggestion ?

**Comment by chiefy** [comment]
@balagcontus remove `AWS_ES_CREDS` and `AWS_DEFAULT_REGION`

**Comment by balagcontus** [comment]
Thanks for your response. 

I have removed both from ENV file and run again node index.js , still getting error, please suggestionto complete, I am struggled for 2 weeks. 

ENV file and error screenshot. 


 
![screenshot_2016-04-01_20-10-01](https://cloud.githubusercontent.com/assets/5197274/14210292/03fc3554-f846-11e5-9b73-1a7a3774e466.png)
![screenshot_2016-04-01_20-09-51](https://cloud.githubusercontent.com/assets/5197274/14210295/06ad6656-f846-11e5-9453-20188fe996b9.png)

**Comment by chiefy** [comment]
@balagcontus are you using opbeat? you have `OPBEAT_ACTIVE=true`, the 403 might be from opbeat since it doesn't look like you have set any keys for opbeat. Try setting that to `false`

**Comment by balagcontus** [comment]
@chiefy Thanks for response. 

I have set to false and still same error.  please check the screen and different solution for seneca fatal error. 

![screenshot_2016-04-01_20-24-41](https://cloud.githubusercontent.com/assets/5197274/14210722/e1b2164c-f847-11e5-98be-797ba8d834d0.png)

**Comment by chiefy** [comment]
@balagcontus only thing I can think of is changing to `http` (not `https`, if that doesn't work, it might be a bug in our plugin.

**Comment by balagcontus** [comment]
@chiefy 

Thanks for your response. 

I have changed aws elasticsearch https to http but still getting same error. please look attached screenshot. 

![image](https://cloud.githubusercontent.com/assets/5197274/14231564/e1ca36aa-f9a5-11e5-9154-a3cc6045decf.png)

I waiting for your reply.

**Comment by blainsmith** [comment]
This has been fixed with the 2.0.0 update.
